platforms/r3-corda/configuration/roles/create/certificates/node/tasks/main.yaml

```diff
@@ -112,6 +112,14 @@
     VAULT_ADDR: "{{ vault.url }}"
     VAULT_TOKEN: "{{ vault.root_token }}"
 
+- name: "Write cordapps credentials to vault"
+  shell: |
+    vault kv put {{ component_name }}/credentials/userpass user="{{ cordapps_details.username }}" pass="{{ cordapps_details.password }}"
```

Comment:
Use a different kv, more relevant to what the username/password id for
like  "credentials/cordapps", "repo_username", "repo_password". Add corresponding changes in the charts.